app/upgrades.go

```diff
@@ -194,6 +196,7 @@ var upgrades = map[string]appUpgrade{
 
 			return vm, nil
 		},
+		Added: []string{ibcratelimit.ModuleName},
```

Comment:
Can we put these `Added` entries above the `Handler` please (for both upgrades)? I know we've had it below in the past, but I'd kind of like to change that. I feel like it makes more sense above it since it's things being added and is just a slice of strings.